project.php

```diff
@@ -1971,54 +1971,30 @@ function sr_echo_time_form($label, $min_days, $max_days, $default_days, $extend
 
 function echo_smoothreading_options($project)
 {
-    global $projects_url;
-
     $smooth_dir = "$project->dir/smooth";
-    $project_url = "$projects_url/$project->projectid";
-    $smooth_url = "$project_url/smooth";
+    $smooth_url = "$project->url/smooth";
     echo "<li class='list-head'>", _("Download a Smooth Reading file");
     echo "<ul>";
     echo_file_downloads(glob("$smooth_dir/*.txt"), $smooth_url);
 
-    // download zipped html
-    $html_files = glob("$smooth_dir/*.{htm,html}", GLOB_BRACE);
-    if($html_files)
+    // zipped htm(l) file
+    $zip_files = glob("$smooth_dir/*.zip");
+    foreach(glob("$smooth_dir/*.zip") as $zip_file) // only one
```

Comment:
What does "only one" mean? What happens if/when there is more than one for some unexpected reason?

Reply:
This comment was meant to indicate that we knew that there was not more than one because we made only on on upload so it was shorter to write "foreach ..." than testing if there were any and then selecting one, but as you say there could be more for an unexpected reason such as if the PP had uploaded another zip file inside his main zip. This does raise some other issues so we need to discuss it further. I'll mention it on Slack.